### DISCUSSION TITLE
Search in Folder View

### DISCUSSION THREAD

**Discussion by lacrimal** [photoprism/photoprism#2864] [Q&A] [answered]
There is something that is bothering me from a while - why there is no Search available in folder view ?

I know that I can do in search ```folder:MyFolder label:my label```but why I can't use ```label:mylabel``` inside folder ```MyFolder```?

**Comment by graciousgrey** [comment, marked as answer]
We already have a feature request for this: https://github.com/photoprism/photoprism/issues/651 :)

**Reply by lacrimal** [reply]
Indeed, topic was to complex to show up in my search :-)
